src/app/profile/yourGiving/stopStartRecurringGifts/restartGift/step0/addUpdatePaymentMethod.component.js

```diff
@@ -0,0 +1,86 @@
+import angular from 'angular';
```

Comment:
@OzzieOrca , I wrote it from scratch at first. And then I found out you had something similar. I thought I could reuse it but then I saw that there is a difference in bindings. Sp I kept that component and reused payment method selector for updating.